src/test/regress/sql/multi_prepare_plsql.sql

```diff
@@ -247,6 +247,104 @@ SELECT plpgsql_test_2();
 -- SELECT plpgsql_test_6(155);
 -- SELECT plpgsql_test_6(1555);
 
+-- test router executor parameterized PL/pgsql functions
+CREATE TABLE temp_table (
+	key int,
+	value int
+);
+SELECT master_create_distributed_table('temp_table','key','hash');
+SELECT master_create_worker_shards('temp_table',4,1);
+
+CREATE OR REPLACE FUNCTION no_parameter_insert() RETURNS void as $$
+BEGIN
+	INSERT INTO temp_table (key) VALUES (0);
+END;
+$$ LANGUAGE plpgsql;
+
+SELECT no_parameter_insert();
```

Comment:
> How about reducing error verbosity to ERROR?

That's a good idea, but still, job ids in the ERROR message can change because we run multi_prepare_sql and multi_prepare_plsql in parallel. We can run them sequentially, but the comment here notes that running them decreases regression test time 5 seconds.
https://github.com/citusdata/citus/issues/346#issue-134118154

What do you think? Do you have any preference?